datastore/db/bytebase/FertiScan/get_inspection/get_sub_label.sql

```diff
@@ -1,33 +1,30 @@
 CREATE OR REPLACE FUNCTION "fertiscan_0.0.14".get_sub_label_json(label_info_id uuid)
- RETURNS jsonb
- LANGUAGE plpgsql
+RETURNS jsonb
+LANGUAGE plpgsql
 AS $function$
-declare
+DECLARE
 	result_json jsonb;
-    fr text[] ;
-    en text[] ;
 BEGIN
     SELECT jsonb_object_agg(
                 sub_type.type_en, 
                 jsonb_build_object(
-				    'en', COALESCE(sub_label_data.texts_en, array[]::text[]),
-				    'fr', COALESCE(sub_label_data.texts_fr, array[]::text[])
-				)			
+				    'en', COALESCE(NULLIF(sub_label_data.texts_en, array[]::text[]), NULL),
```

Comment:
This doesn't work with our standard for the getters. It must return an empty array if there are no entry for this object. I dont think you need to change the getter

Reply:
Yes, that was not intended. Sometimes gpt gets a bit overzealous. Fixed.